### DISCUSSION TITLE
Firebase functions

### DISCUSSION THREAD

**Discussion by snorreks** [vercel/turborepo#640] [Help] [answered]
Is anyone using firebase functions with turborepo? If so, how are you importing packages?

**Comment by dmurawsky** [comment]
I am and I'm having trouble with it:
<img width="1029" alt="Screen Shot 2022-03-14 at 2 48 39 PM" src="https://user-images.githubusercontent.com/3521396/158240566-8f5498fe-f5da-477d-9810-cdc6071a8e0c.png">
<img width="989" alt="Screen Shot 2022-03-14 at 2 49 02 PM" src="https://user-images.githubusercontent.com/3521396/158240618-7fc73447-127d-414c-ae8c-f5f760a88a29.png">

**Comment by romshiri** [comment]
Did you guys find and answer?

**Comment by Hacksore** [comment, marked as answer]
A PR for getting a simple firebase example added has been submitted https://github.com/vercel/turborepo/pull/1966
